### ISSUE TITLE
Initial window too small on 4k display

### ISSUE THREAD

**Issue by Svetloslav** [tomasklaen/uosc#872]
One of the updates after the official version 5.2, made the initially opened empty window probably half the size compared to the previous empty window in v5.2. That is on a 4k display. Before it was just the right size, while now it is too small and first thing one has to do is to resize (extra, pointless step). Granted it does not affect actual playback of videos, since those resize according to the resolution of the video anyways, but it is visually irritating regression especially if one wants to drag and drop a file on top for a quick playback.

**Comment by hooke007** [comment]
What does it have to do with UOSC?

**Comment by hooke007** [comment]
https://github.com/mpv-player/mpv/commit/74b4c3c531bd454484ed5d04c9aad0bc531ccd4b
optimize your mpv conf instead.

**Comment by Solarunit** [comment]
in mpv config you can set `geometry` setting, e.g.:

`geometry=60%`

**Comment by Svetloslav** [comment]
Thank you @Solarunit, I solved it yesterday by visiting the link provided by @hooke007, where it was clear that it was the development of mpv, who made a recent change. Reverting that change, fixed the issue by applying:
hidpi-window-scale=yes
However, your suggestion might be more universal, since if one switches to a non-4k display everything will probably look out of proportion with that hidpi setting. I'll try geometry instead.

**Comment by Svetloslav** [comment]
Nope, geometry has no effect whatsoever. I'll stick to my original solution.
